azure-function/dotnet/Models/Vehicle.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace GetBusData.Models;
+
+public sealed class Vehicle
+{
+    [JsonProperty("trip")]
+    public Trip? Trip { get; set; }
+
+    [JsonProperty("vehicle")]
+    public VehicleId? VehicleId { get; set; }
+
+    [JsonProperty("position")]
```

Comment:
I don't think this `JsonProperty(...)` is necessary because the function app applies camelCasing by default.